src/Core/Localization/Specification/Number.php

```diff
@@ -325,13 +325,15 @@ protected function validateData()
     }
 
     /**
-     * @deprecated https://github.com/PrestaShop/PrestaShop/issues/13168
+     * @deprecated
      *
      * @param int $maxFractionDigits
+     *
+     * @throws LocalizationException
      */
     public function setMaxFractionDigits($maxFractionDigits)
     {
-        $this->maxFractionDigits = $maxFractionDigits;
+        throw new LocalizationException('Class Number is immutable. Please define maxFractionDigits from the constructor.');
```

Comment:
Since this setter now throws an exception this could be considered as a BC break anyway, so my first comment seems realistic
Maybe you could even drop this setter?

Reply:
Method removed.